data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/v1/beacon/PostAttesterSlashingIntegrationTest.java

```diff
@@ -65,7 +66,9 @@ public void shouldReturnSuccessWhenRequestBodyIsValid() throws Exception {
     final AttesterSlashing schemaSlashing = new AttesterSlashing(slashing);
 
     when(attesterSlashingPool.add(slashing))
-        .thenReturn(SafeFuture.completedFuture(InternalValidationResult.ACCEPT));
+        .thenReturn(
+            SafeFuture.completedFuture(
+                InternalValidationResult.create(ValidationResultCode.ACCEPT)));
```

Comment:
nit: I wonder if it's worth having a static `InternalValidatationResult.ACCEPT` - saves a bunch of boiler plate code and creating objects.